src/node/data_store/tests.rs

```diff
@@ -161,10 +161,31 @@ async fn delete() -> Result<()> {
             value: data.clone(),
         };
         data_store.put(the_data).await?;
+
+        let mut used_space = data_store.total_used_space().await;
+
+        while used_space != *size {
+            used_space = data_store.total_used_space().await;
+            tokio::time::sleep(std::time::Duration::from_millis(200)).await;
+        }
```

Comment:
```suggestion
        loop {
            let used_space = data_store.total_used_space().await;
            if used_space == *size {
                break;
            }
            tokio::time::sleep(std::time::Duration::from_millis(200)).await;
        }
```
No mut, only one call. What do you think?